include/odp/api/spec/ipsec.h

```diff
@@ -263,6 +270,11 @@ typedef struct odp_ipsec_capability_t {
 	 *  packets */
 	odp_bool tfc_padding_truncate;
 
+	/**
+	 * Support dropping dummy packets
+	 */
+	odp_support_t dummy_packets_drop;
```

Comment:
`tfc_dummy_packets_drop` would be clearer.

Reply:
ok, updated.